vidarrtest-regression.json.in

```diff
@@ -14,6 +14,7 @@
                 "type": "EXTERNAL"
             },
             "rsem.outputFileNamePrefix": "OCT_011325_Mo_P_WT_OCT-01-1325-TS",
+            "rsem.reference": "hg38",
             "rsem.runRsem.jobMemory": null,
             "rsem.runRsem.modules": "rsem/1.3.3 hg38-rsem-index/1.3.0",
             "rsem.runRsem.rsemIndexDir": "$HG38_RSEM_INDEX_ROOT/hg38_random_rsem",
```

Comment:
since we moved the code for setting assembly-specific parameters (modules and path to indices) to wdl these can be removed from regression testing

Reply:
removed.